lib/active_resource/connection.rb

```diff
@@ -31,11 +31,12 @@ def requests
 
     # The +site+ parameter is required and will set the +site+
     # attribute to the URI for the remote resource service.
-    def initialize(site, format = ActiveResource::Formats::JsonFormat)
+    def initialize(site, format: ActiveResource::Formats::JsonFormat, logger: nil)
```

Comment:
We can't change `format` here without a deprecation cycle. I think we can add logger without changing format.

Reply:
Good point. I'll change it back.